js/coinsbit.js

```diff
@@ -0,0 +1,1103 @@
+        }
+        const price = this.safeString (order, 'price');
+        const orderSize = this.safeString (order, 'amount');
+        const remaining = this.safeString (order, 'left');
+        const filledSize = this.safeString (order, 'dealStock');
+        const cost = this.safeString2 (order, 'dealMoney', 'deal');
+        const fee = this.safeString2 (order, 'dealFee', 'fee');
+        const marketId = this.safeString (order, 'market');
+        if (market === undefined) {
+            market = this.safeMarket (marketId, market, '_');
+        }
+        const symbol = this.safeString (market, 'symbol');
+        const quoteCurrency = market['quote'];
+        // <<<TODO>>> The below three lines are under question
+        // const feeCoeffTaker = this.safeString (order, 'takerFee');
+        const feeCoeffMaker = this.safeString (order, 'makerFee');
+        const willBeFee = Precise.stringMul (cost, feeCoeffMaker); // <<<TODO>> idk if the below 'willBeFee' & 'isFee' implementations make an useful attempt to calculate the actual fee (when using createOrder for limit order, which wasn't executed at market, and thus, has to be executed still). For example, when submiting limit-order (and it's not filled immediately as taker), then `dealFee` returns 0. however, it is still possible to calculate the 'dealFee' that will be when order will be triggered. (so, for limit orders its seems the only way to calculate the fee). however, I'll remove if this is not useful
+        return this.safeOrder2 ({
+            'id': id,
+            'clientOrderId': undefined, // <<<TODO>>> should it be here?
+            'timestamp': timestamp,
+            'datetime': this.iso8601 (timestamp),
+            'lastTradeTimestamp': undefined,
+            'status': undefined,
+            'symbol': symbol,
+            'type': orderType,
+            'timeInForce': undefined,
+            'side': side,
+            'price': price,
+            'stopPrice': undefined,
+            'average': undefined,
+            'amount': orderSize,
+            'cost': cost,
+            'filled': filledSize,
+            'remaining': remaining,
+            'fee': {
+                'cost': fee,
+                'currency': quoteCurrency,
+                'will_cost': willBeFee,
+            },
+            'trades': undefined,
+            'info': order,
+        }, market);
+    }
+
+    async fetchOrderTrades (id, symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        const request = {};
+        request['orderId'] = id;
+        if (limit !== undefined) {
+            request['limit'] = limit;
+        }
+        const response = await this.privatePostAccountTrades (this.extend (request, params));
+        // {
+        //     success: true,
+        //     message: '',
+        //     result: {
+        //       offset: '0',
+        //       limit: '50',
+        //       records: {
```

Comment:
This response sample looks wrong to me. I would expect an array in square brackets `[]` here.